src/ansible/roles/keycloak/tasks/main.yml

```diff
@@ -23,9 +27,10 @@
     create_home: yes
 
 - name: Download software zipfile
-  get_url:
-    url: "{{ keycloak_url }}"
-    dest: /tmp/keycloak.tgz
+  command: curl -L {{ keycloak_url }} -o /tmp/keycloak.tgz
+#  get_url:
+#    url: "{{ keycloak_url }}"
+#    dest: /tmp/keycloak.tgz
```

Comment:
I think you can just remove this comment as the change will be visible in git history.

Reply:
done